pkg/project/project.go

```diff
@@ -247,9 +248,15 @@ func (p *Project) Cache() string { return p.projectfile.Cache }
 
 // Namespace returns project namespace
 func (p *Project) Namespace() *Namespaced {
-	commitID, err := localcommit.GetCompatible(p)
+	commitID, err := localcommit.Get(p.Dir())
```

Comment:
It just occurred to me we should not be exposing the commit ID through here at all. Could you see if it's easy to drop this from here or if not file a followup story? Effectively the project package should have zero awareness of localcommit, only of legacy commit.

Reply:
Sure. After looking through the codebase, I cannot find a legitimate use for it anymore.